charts/devlake/templates/deployments.yaml

```diff
@@ -122,12 +122,14 @@ spec:
       securityContext:
       {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if not .Values.mysql.useExternal }}
```

Comment:
I think it needs to be also check when both values of `.Values.mysql.useExternal` and `.Values.pgsql.useExternal` is not true?

Reply:
yes, you are right.  Let me make that change